app/assets/javascripts/modules/coronavirus-landing-page.js

```diff
@@ -32,15 +30,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
   }
 
-  CoronavirusLandingPage.prototype.addAccordionOpenAllTracking = function ($element) {
-    $element.find('.gem-c-accordion__open-all').on('click', function (e) {
-      var expanded = $(e.target).attr('aria-expanded') === 'true'
-      var label = expanded ? 'Expand all' : 'Collapse all'
-      var action = expanded ? 'accordionOpened' : 'accordionClosed'
-      var options = { transport: 'beacon', label: label }
+  CoronavirusLandingPage.prototype.addAccordionOpenAllTracking = function () {
+    var accordions = this.module.querySelectorAll('.gem-c-accordion')
+    for (var i = 0; i < accordions.length; i++) {
+      accordions[i].addEventListener('click', function (e) {
+        var clicked = e.target.closest('button')
```

Comment:
`closest()` will return `null` if an element matching the selector is not found. I understand such an element should in theory always exist in the context of the accordion component, but in a scenario where it is not found, `clicked.classList` would throw an `Uncaught TypeError`.
So basically what I'm saying is, should this conditional  be changed to `if (clicked && clicked.classList.contains('gem-c-accordion__open-all')) {}` 😄

Reply:
Actually I can replicate this hypothetical issue in a browser too (recreate it by clicking inside the accordion but outside the `gem-c-accordion__open-all` link)

https://user-images.githubusercontent.com/7116819/140968420-aa0d4fe3-5015-48a6-8229-970e11c93ea1.mov

Reply:
Great spot, thanks. Have updated.